cpg_infra/billing_aggregator/aggregate/seqr.py

```diff
@@ -161,8 +161,13 @@ def get_finalised_entries_for_batch(
                 currency_conversion_rate, raw_cost=raw_cost
             )
             raw_usage = job['resources'].get(batch_resource, 0)
+
+            # Distribute the remaining cost across all datasets proportionately
             for dataset, (fraction, dataset_size) in prop_map.items():
-                # Distribute the remaining cost across all datasets proportionately
+                # 2023-03-07 mfranklin: I know this key isn't unique, but to avoid
+                # issues with changing the resource_id again, we'll only use the
+                # batch_id as the key as it's sensible for us to assume that all the
```

Comment:
Here and below, the comment says "only use the batch_id", but the implementation actually includes the `job_id` too.